modules/hcp-dr-oadp-restore-new-mgmt.adoc

```diff
@@ -0,0 +1,302 @@
++
+[source,terminal]
+----
+$ oc --kubeconfig <restore_management_kubeconfig> \
+  get nodepool -n <hosted_cluster_namespace> 
+----
++
+.Example output
+[source,terminal]
+----
+NAME       CLUSTER    DESIRED NODES   CURRENT NODES   AUTOSCALING   AUTOREPAIR   VERSION   UPDATINGVERSION   UPDATINGCONFIG   MESSAGE
+hosted-0   hosted-0   3               3               False         False        4.17.11   False             False
+----
+
+.. Optional: To ensure that no conflicts exist and that the new management cluster has continued functionality, remove the `HostedCluster` resources from the backup management cluster by completing the following steps:
+
+... In the management cluster that you backed up from, in the `ClusterDeployment` settings, set the `spec.preserveOnDelete` parameter to `true` by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <backup_management_kubeconfig> patch \
+  -n <hosted_control_plane_namespace> \
+  ClusterDeployment/<hosted_cluster_name> -p \
+  '{"spec":{"preserveOnDelete":'true'}}' \
+  --type=merge
+----
++
+This step ensures that the hosts are not deprovisioned.
+
+... Delete the machines by running the following commands:
++
+[source,terminal]
+----
+$ oc --kubeconfig <backup_management_kubeconfig> patch \
+  <machine_name> -n <hosted_control_plane_namespace> -p \
+  '[{"op":"remove","path":"/metadata/finalizers"}]' \
+  --type=merge
+----
++
+[source,terminal]
+----
+$ oc --kubeconfig <backup_management_kubeconfig> \
+  delete machine <machine_name> \
+  -n <hosted_control_plane_namespace>
+----
+
+... Delete the `AgentCluster` and `Cluster` resources by running the following commands:
++
+[source,terminal]
+----
+$ oc --kubeconfig <backup_management_kubeconfig> \
+  delete agentcluster <hosted_cluster_name> \
+  -n <hosted_control_plane_namespace>
+----
++
+[source,terminal]
+----
+$ oc --kubeconfig <backup_management_kubeconfig> \
+  patch cluster <cluster_name> \
+  -n <hosted_control_plane_namespace> \
+  -p '[{"op":"remove","path":"/metadata/finalizers"}]' \
+  --type=json
+----
++
+[source,terminal]
+----
+$ oc --kubeconfig <backup_management_kubeconfig> \
+  delete cluster <cluster_name> \
+  -n <hosted_control_plane_namespace> 
+----
+
+... If you use {rh-rhacm-title}, delete the managed cluster by running the following commands:
++
+[source,terminal]
+----
+$ oc --kubeconfig <backup_management_kubeconfig> \
+  patch managedcluster <hosted_cluster_name> \
+  -n <hosted_cluster_namespace> \
+  -p '[{"op":"remove","path":"/metadata/finalizers"}]' \
+  --type=json
+----
++
+[source,terminal]
+----
+$ oc --kubeconfig <backup_management_kubeconfig> \
+  delete managedcluster <hosted_cluster_name> \
+  -n <hosted_cluster_namespace>
+----
+
+... Delete the `HostedCluster` resource by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <backup_management_kubeconfig> \
+  delete hostedcluster -n \
```

Comment:
Not sure if you need the `\` on line 300

Reply:
I think it's required because it's part of a long command, and there's a line after it.